### DISCUSSION TITLE
Electrical Efficiency

### DISCUSSION THREAD

**Discussion by vault-thirteen** [ByteWelder/Decktility#2] [General]
Hello there !

I am dreaming of a PDA similar to yours. 
Me thinks that 18650 batteries are the best choice for PDA in this time of being.

What is the total electrical efficiency of the intermediary circuit boards between the 18650 batteries and the Raspberry Pi board ?

Thank you.

**Comment by KenVanHoeylandt** [comment]
I'm afraid I cannot give you exact numbers on efficiency, only on power draw and details on specific components. I don't have an electrical engineering background, so my knowledge on the topic is very limited.

The reason that I picked a 2S (2 cell) setup is due to increased efficiency: most step up converters that I've used get hot, while my step down converters don't. My understanding is that this heat is lost energy and thus implies lower efficiency.

That's also why I believe that the USB BMS efficiency isn't great. It gets hot while charging. It needs to step up `5 V` to `8.4 V`. A better BMS could solve the trick, as USB PD or QC could ask the charger for a higher voltage. Sadly, I could not find such a BMS with USB-C. I might look into it at a later point, but it's not on my short-term todo list.

However, when the device is powered on (and not charging), the battery voltage is stepped down to `5 V`. It's likely close to 90% efficient range and perhaps even higher. There is no perceptible heat coming off of it.

I'm not an electrical engineer, so the best I can do towards providing you real numbers is giving you some of the component datasheets. For the power switching, I'm using:

- [IRF5305](https://www.infineon.com/dgdl/irf5305spbf.pdf?fileId=5546d462533600a4015355e378101995) mosfet
- [PC817](https://pdf1.alldatasheet.com/datasheet-pdf/view/43376/SHARP/PC817C.html) opto-coupler
- SMD LED (0805 size, I think)

Efficiency is always on the back of my mind though. Mainly due to heat generation, because heat escapes where lots of power is lost/converted. Heat requires cooling, and cooling is tricky. The tiny 2006 fan easily draws 0.3-0.6W all by itself.
On a similar note: The power draw of the Arduino Nano is about `0.2 W`, without peripherals attached. The RP2040 Zero that I'm replacing it with uses about `0.15 W` without perhipherals and when clocked down to `50 MHz`.
